src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -13,34 +13,377 @@ import spray.http.StatusCodes
 /**
  * Created by dvoet on 2/4/16.
  */
-case class AttributeRecord(id: UUID,
+case class EntityAttributeRecord(id: Long,
+                           entityId: Long,
                            name: String,
                            valueString: Option[String],
                            valueNumber: Option[Double],
                            valueBoolean: Option[Boolean],
-                           valueEntityRef: Option[UUID],
+                           valueEntityRef: Option[Long],
                            listIndex: Option[Int])
 
+case class WorkspaceAttributeRecord(id: Long,
+                                    workspaceId: UUID,
+                                    name: String,
+                                    valueString: Option[String],
```

Comment:
You might consider:
```scala
type ValueString = Option[String]
```
etc. to get some reuse here. Probably not worth doing but wanted to offer the suggestion anyway.